activity_browser/app/bwutils/multilca.py

```diff
@@ -141,25 +168,85 @@ def get_all_metadata(self):
 
 
 class Contributions(object):
-    """Contribution Analysis built on top of the Multi-LCA class."""
+    """Contribution Analysis built on top of the Multi-LCA class.
+
+    This class requires instantiated MLCA and MetaDataFrame objects.
```

Comment:
what is the right name now MetaDataFrame or MetaDataStore ? I have seen both now :-)

Reply:
Whoops, you're right, have been using them interchangeably, will correct to MetaDataStore.